updater/lib/dependabot/dependency_change.rb

```diff
@@ -165,24 +165,32 @@ def matches_existing_pr?
       if grouped_update?
         # We only want PRs for the same group that have the same versions
         job.existing_group_pull_requests.any? do |pr|
+          directories_in_use = pr["dependencies"].all? { |dep| dep["directory"] }
```

Comment:
hmm would a dependency have a directory if it was removed?

Reply:
Yes, it always is added during DependencyChange initialization: https://github.com/dependabot/dependabot-core/blob/1cec7cdfa773a29a81dd3ce15d512314dd896392/updater/lib/dependabot/dependency_change.rb#L200-L205

And the serializer in the API also always adds it. I won't link to that here but you can find it 😃